src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -26,13 +26,18 @@ class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
     private $wasSkipped = array();
     private $isSkipped = array();
 
-    public function __construct(array $extraClockMockedNamespaces = array())
+    public function __construct(array $extraClockMockedNamespaces = array(), array $extraDnsMockedNamespaces = array())
```

Comment:
That does not scale well. I think we need to think about another way to inject those mocked configs.